pyproject.toml

```diff
@@ -3,12 +3,15 @@
 line-length = 100
 # Assume Python 3.10
 target-version = "py310"
+# Source
+src = ["bin"]
 
-[tool.ruff.lint]
-# Ignore star-imports
-# TODO #102: This is definitely fixable, but for now, postponing the clean-up of all imports
-ignore = ["F403", "F405"]
-
+[tool.ruff.lint.isort]
+# organize imports in two blocks, non local and local
+section-order = ["standard-library", "first-party"]
+default-section = "standard-library"
```

Comment:
@mzuenni Do you have a strong opinion against the default ordering (`["standard-library", "third-party", "first-party"]`)? I think I would prefer the default ordering 🙂

Reply:
Fixed in 45c7f0421bde9ba5394234eced854dc8889cb78e :smile: